src/VSSTypeSanitizer.cpp

```diff
@@ -37,14 +37,14 @@ void checkNumTypes(jsoncons::json &meta, jsoncons::json &val )
     try {
         cval = val.as<T>();
     }
-    catch(...) {
```

Comment:
Is it clever to change this? I am not sure what kind of exceptions  are thrown by the underlying jsoncons (and other standard libs it uses on different OS), and wanted to be sure here, even if it is bad style :)

Theoretically the change can miss excpetions, that are not derived std::exception
https://stackoverflow.com/questions/315948/c-catching-all-exceptions/24142104

Maybe the risk is small, but it was a conscious discussion to put "..."

Opinions?

Reply:
I will also add this catch